src/fides/api/models/privacy_experience.py

```diff
@@ -432,10 +406,10 @@ def new_experience_needed(
         )
 
         privacy_center_notices: Query = get_privacy_notices_by_region_and_component(
-            db, region, ComponentType.privacy_center
+            db, [region.value], ComponentType.privacy_center
         )
         overlay_notices: Query = get_privacy_notices_by_region_and_component(
-            db, region, ComponentType.overlay
+            db, [region.value], ComponentType.overlay
```

Comment:
Same thing here, haven't thought through the consequences, but if a PrivacyExperience object does not exist for a given region x component, then  notices will not be shown.